### DISCUSSION TITLE
Get-PnPSiteTemplate fails with 'scope' error

### DISCUSSION THREAD

**Discussion by GHammar** [pnp/powershell#4136] [Q&A]
I want to download a SharePoint Online template as I have done in the past. I set a couple of variables and connect as my regular user using

Connect-PnPOnline -Url $SiteURL -Interactive

The connect succeeds. Then I run 

Get-PnPSiteTemplate -Out ($SchemaXMLPath) -PersistBrandingFiles -Force 

to download the template. The download starts and fails halfway through with this error:

Message:  AADSTS70011: The provided request must include a 'scope' input parameter. The provided value for the input parameter 'scope' is not valid. The scope https://graph.microsoft.com openid profile offline_access is not valid. The scope format is invalid. Scope must be in a valid URI form <https://example/scope> or a valid Guid <guid/scope>. 

This is the output from Get-Module -Name "Pnp.PowerShell" -ListAvailable

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Manifest   2.5.40     nightly    PnP.PowerShell                      Desk      {Add-PnPAdaptiveScopeProperty, Add-PnPE…
Manifest   2.4.71     nightly    PnP.PowerShell                      Desk      {Add-PnPAdaptiveScopeProperty, Add-PnPE…
Manifest   2.4.44     nightly    PnP.PowerShell                      Desk      {Add-PnPAdaptiveScopeProperty, Add-PnPE…
Manifest   2.2.0                 PnP.PowerShell                      Desk      {Add-PnPAdaptiveScopeProperty, Add-PnPP…

I'd appreciate any suggestions as to next steps.

**Comment by ConaxLiu** [comment]
I have the same issue when I run Get-PnPSiteTemplate on a customer's tenant and I am one of the site collection administrators. The cmd-let errored at [Web Settings]. If I configure a configuration file to exclude 'Web Settings' then the cmd-let could execute without error.

Running Get-PnPSiteTemplate on my own test tenant have no issues at all.

Running in Visual Studio Code.

Directory: D:\OneDrives\OneDrive - My Organisation Name\Documents\PowerShell\Modules
Manifest       2.9.0       PnP.PowerShell    Core         {Add-PnPAdaptiveScopeProperty, Add-PnPEntraIDGroupMember,...
Manifest       2.5.0       PnP.PowerShell    Core         {Add-PnPAdaptiveScopeProperty, Add-PnPEntraIDGroupMember,...

Directory: C:\Program Files\PowerShell\Modules
Manifest       2.5.0       PnP.PowerShell    Core         {Add-PnPAdaptiveScopeProperty, Add-PnPEntraIDGroupMember,… 

PS Version: 7.4.3
PS Edition: Core
